src/main/java/org/jenkinsci/plugins/workflow/support/steps/input/InputStepExecution.java

```diff
@@ -283,7 +283,7 @@ private void postSettlement() {
     }
 
     private boolean canCancel() {
-        return !Jenkins.getInstance().isUseSecurity() || getRun().getParent().hasPermission(Job.CANCEL);
+        return !Jenkins.getActiveInstance().isUseSecurity() || getRun().getParent().hasPermission(Job.CANCEL);
```

Comment:
I can never remember what the right method to use here is - but it looks like as of core 2.98, you'd want to use `Jenkins.get()` here - `Jenkins.getActiveInstance()` is deprecated (and has been for a long time, apparently - not deprecated in 1.642.4, which we're using here, but deprecated in 2.7.3...) and now as of 2.98, `Jenkins.getInstance()` is deprecated too. 

So, bugger it. 1.642.4 has `Jenkins.getInstance()` and `Jenkins.getActiveInstance()`, both non-deprecated. 2.7.3 has both plus `Jenkins.getInstanceOrNull()`, with `Jenkins.getActiveInstance()` now deprecated. And 2.98 has all three plus `Jenkins.get()`, with `Jenkins.getActiveInstance()` and `Jenkins.getInstance()` both deprecated. That's a confusing pile of deprecation and methods, so just go with what works. =)